meinberlin/apps/cms/models.py

```diff
@@ -73,6 +73,7 @@ class HomePage(Page):
         ('columns_text', cms_blocks.ColumnsBlock()),
         ('projects', cms_blocks.ProjectsWrapperBlock()),
         ('activities', actions_blocks.PlatformActivityBlock()),
+        ('accordion', cms_blocks.DocsBlock())
     ])
```

Comment:
Is it ok to call it accordion when block has different name?

Reply:
yes